src/main/java/ru/spbstu/controller/VcfController.java

```diff
@@ -27,4 +31,22 @@
             return ResponseEntity.status(HttpStatus.UNPROCESSABLE_ENTITY).body("Incorrect vcf file");
         }
     }
+
+    @PostMapping("/getAnnotatedVariants")
+    @ResponseBody
+    public AnnotatedVariantList getAnnotatedVariants(@RequestBody VariantList requestedVariantList) {
+        List<Variant> requestedVariants = requestedVariantList.getVariants();
```

Comment:
Delegate internal logic to Service.

Reply:
Internal logic was delegated to VariantService